spec/lib/heart-check/checks/watch_file_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
```

Comment:
We are including in all specs in https://github.com/locaweb/heartcheck/blob/master/.rspec

You could remove this line